### PR TITLE
create routing and pagination

### PR DESCRIPTION
**Pull request by yurii-kdz**
create routing and pagination

### CODE REVIEW

File: my-app/src/App.js [hunk 1]
```diff
@@ -2,13 +2,18 @@ import React from 'react'
...
+    BrowserRouter as Router,
+} from "react-router-dom";
 
 const App = () => {
 
     return (
         <Container disableGutters>
-            <Header />
-            <Body />
+            <Router>
```
Comment (ArtGarm) [inline my-app/src/App.js:14]: why no switch inside the router?

File: my-app/src/components/Body/Content/BlogView/BlogView.jsx [hunk 2]
```diff
@@ -4,33 +4,50 @@ import TableCell from '@material-ui/core/TableCell';
...
+    const urlSearchParams = new URLSearchParams(location.search);
+    const page = parseInt(urlSearchParams.get('page') || 1);
+
+    return page > 1 ? page : 1;
+};
 
 const BlogView = (props) => {
     const classes = useStyles(props);
 
+    const location = useLocation();
```
Comment (ArtGarm) [inline my-app/src/components/Body/Content/BlogView/BlogView.jsx:28]: why not enough useParams?

File: my-app/src/components/Body/Content/Content.jsx [hunk 3]
```diff
@@ -1,9 +1,22 @@
...
+    Route,
+} from "react-router-dom";
 
 const Content = () => {
-    return(
-        <BlogView/>
+    return (
+        <Switch>
+            <Route exact path="/" component={
+                () => <BlogView/>
```
Comment (ArtGarm) [inline my-app/src/components/Body/Content/Content.jsx]: why the component is arrow function?

File: my-app/src/components/Body/Content/Content.jsx [hunk 4]
```diff
@@ -1,9 +1,22 @@
...
+            }>
+            </Route>
+            <Route exact path="/createView">
+                <CreateView/>
+            </Route>
```
Comment (ArtGarm) [inline my-app/src/components/Body/Content/Content.jsx:14]: use route component in same style

File: my-app/src/components/Body/Content/CreateView/style.js [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+    dot: {
+        display: "block",
+        borderRadius: "50%",
+        marginRight: theme.spacing(0.5),
+        background: theme.palette.grey.A700,
+        height: "5px",
+        width: "5px"
+    },
+    ul: {
+        fontSize: "16px",
```
Comment (ArtGarm) [inline my-app/src/components/Body/Content/CreateView/style.js]: use from typography

File: my-app/src/components/Body/Menu/mockup.js [hunk 6]
```diff
@@ -12,18 +12,18 @@ import Coffee from "../../../icons/Coffee";
...
 const data = {
     menuItems : [
-        {text: "View site", icon: <Home viewBox="0 0 24 24"/>},
-        {text: "Create page", icon: <FilePlus viewBox="0 0 24 24"/>},
-        {text: "Blog articles", icon: <PenTool viewBox="0 0 24 24"/>},
-        {text: "Files", icon: <Image viewBox="0 0 24 24"/>},
-        {text: "Users", icon: <Users viewBox="0 0 24 24"/>},
-        {text: "Subscriptions", icon: <Zap viewBox="0 0 24 24"/>},
-        {text: "Archived pages", icon: <Trash viewBox="0 0 24 24"/>}
+        {text: "View site", icon: <Home viewBox="0 0 24 24"/>, route: "/"},
```
Comment (ArtGarm) [inline my-app/src/components/Body/Menu/mockup.js:15]: looks bad

**Review by ArtGarm — CHANGES_REQUESTED** [review]

File: my-app/src/components/Body/Content/Content.jsx (hunk 3 above)
Comment (yurii-kdz) [reply my-app/src/components/Body/Content/Content.jsx]: > why the component is arrow function?

if I don't use arrow function, my component doesn`t rerend after pressing on paginate button ¯\_(ツ)_/¯

File: my-app/src/components/Body/Content/Content.jsx [hunk 7]
```diff
@@ -1,9 +1,20 @@
...
+} from "react-router-dom";
 
 const Content = () => {
-    return(
-        <BlogView/>
+    return (
+        <Switch>
+            <Route exact path="/" render={() => <BlogView/>}>
+            </Route>
+            <Route exact path="/createView" render={() => <CreateView/>}>
```
Comment (ArtGarm) [inline my-app/src/components/Body/Content/Content.jsx]: use this format
`<Route exact path="/createView" component={CreateView} />`

this format 
`<Route exact path="/createView" render={() => <CreateView/>}>`
used only if you need to drill some props

**Review by ArtGarm — CHANGES_REQUESTED** [review]

**Review by ArtReactDev — APPROVED** [review]

**Pull request merged**